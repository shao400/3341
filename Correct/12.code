pRogram Input outpUt iF TheN ENDIF BEGIN EnD EOS