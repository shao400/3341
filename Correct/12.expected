ID[pRogram]
ID[Input]
ID[outpUt]
ID[iF]
ID[TheN]
ID[ENDIF]
ID[BEGIN]
ID[EnD]
ID[EOS]
